README.md

```diff
@@ -15,12 +25,10 @@ This is the simple application which simulate how the ATM system works.
 5. Create __jar__ file by running: `jar cfvm atm.jar MANIFEST.mf -C bin/ .`
 
 ### Run the Application
-Type command: `java -jar atm.jar` on your Terminal. And then __*Voila!*__
-
-#### Dummy User
-You can use this account below to try this simulation.
+Type command: `java -jar atm.jar <data_source_filepath>` on your Terminal. 
```

Comment:
And what if I don't pass a file?

Reply:
I have update the ways to running the application doc ae762ee